sdk/communication/Azure.Communication.CallAutomation/src/Models/DtmfTone.cs

```diff
@@ -13,5 +13,75 @@ namespace Azure.Communication.CallAutomation
     [JsonConverter(typeof(EquatableEnumJsonConverter<DtmfTone>))]
     public readonly partial struct DtmfTone
     {
+        /// <summary>
+        /// The Dtmf Tone in character format.
+        /// </summary>
+        public string ToChar()
+        {
+            string originalValue = ToString();
+            return TransferToChar(originalValue);
+        }
+
+        /// <summary>
+        /// Convert the tone from string to char digit like "1", "2", "#".
+        /// </summary>
+        private static string TransferToChar(string value)
```

Comment:
Probably `ConvertToChar` and returning a `char` value is better.

Reply:
changed to converttochar